README.md

```diff
@@ -12,6 +12,7 @@ Build:
 git clone https://www.github.com/alandefreitas/clang-unformat
 cd clang-unformat
 mkdir build
```

Comment:
@alandefreitas, @ryanenzo , this could be further improved, by using modern CMake CLI options. Instead of lines 14:16, better to use
```
cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS=-03
```
and last line then becomes 
```
cmake --build build --target clang-unformat
```

Reply:
Yep. I've been using `-B build` in another library of mine and it's much better: https://alandefreitas.github.io/futures/quickstart/

What we have in this PR is already better than what we had before anyway so I'll merge it and then I'll leave that up to you guys :)